src/business-logic/services/test/execute-action-service.spec.ts

```diff
@@ -0,0 +1,180 @@
+import { ExecuteActionService } from '../execute-action-service'
+import { Action, PartAction, PieceAction } from '../../../model/entities/action'
+import { ActionType } from '../../../model/enums/action-type'
+import { Part, PartInterface } from '../../../model/entities/part'
+import { ConfigurationRepository } from '../../../data-access/repositories/interfaces/configuration-repository'
+import { ActionRepository } from '../../../data-access/repositories/interfaces/action-repository'
+import { RundownService } from '../interfaces/rundown-service'
+import { RundownRepository } from '../../../data-access/repositories/interfaces/rundown-repository'
+import { Blueprint } from '../../../model/value-objects/blueprint'
+import { anyOfClass, anyString, capture, instance, mock, verify, when } from '@typestrong/ts-mockito'
+import { Piece, PieceInterface } from '../../../model/entities/piece'
+
+describe(ExecuteActionService.name, () => {
+  describe(`${ExecuteActionService.prototype.executeAction.name}`, () => {
```

Comment:
Per Anders comment regarding string interpolation, this can also be updated, to not use string interpolation.